ebos/ecloutputblackoilmodule.hh

```diff
@@ -194,15 +194,15 @@ public:
             }
         }
 
-        forceDisableFipOutput_ = EWOMS_GET_PARAM(TypeTag, bool, ForceDisableFluidInPlaceOutput);
+        forceDisableReportsOutput_ = EWOMS_GET_PARAM(TypeTag, bool, ForceDisableReportsOutput);
     }
 
     /*!
      * \brief Register all run-time parameters for the Vtk output module.
      */
     static void registerParameters()
     {
-        EWOMS_REGISTER_PARAM(TypeTag, bool, ForceDisableFluidInPlaceOutput,
+        EWOMS_REGISTER_PARAM(TypeTag, bool, ForceDisableReportsOutput,
                              "Do not print fluid-in-place values after each report step even if requested by the deck.");
```

Comment:
This comment is now outdated. The flag applies to all reports, not just fluid-in-place.